pylib/spinnaker/transform_old_config.py

```diff
@@ -20,6 +20,13 @@
 
 
 class Processor(object):
+  def __collect_environ(self, content):
+    result = {}
+    for match in re.finditer('^([^\s]+?)=([^\n]*)(?:\#.+)?', content,
```

Comment:
`[^\s]` should go to `[\S]` in the first group. Are you trying to cut off `#` comments with the second two groups? If so, I would change `[^\n]` to `[^\n\#]` to not match comments before the non-capturing group.